elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -35,7 +35,7 @@ trait QueryDsl extends DslCommons with SortDsl {
   case class QueryRoot(query: Query,
                        fromOpt: Option[Int],
                        sizeOpt: Option[Int],
-                       sort: Seq[Sort],
+                       sortOpt: Option[Seq[Sort]],
```

Comment:
We need to make this change in a way that doesn't break backwards compatibility (so the arg needs to stay the same).

Can you keep the arg the same and just change toJson to work correctly? I think the fix is `sort.toOption.map(...)` 

`toOption` will make it None if it's empty and Some(list) if it isn't.